#VRML_SIM R2025a utf8
# Describe the functionality of your PROTO here.

EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2025a/projects/appearances/protos/BrushedAluminium.proto"

PROTO Plate1 [
  field SFVec3f    translation    0 0 0
  field SFRotation rotation       0 0 1 0
  field SFString   name           "Plate1"
]
{
  Solid {
    translation IS translation
    rotation IS rotation
    children [
      Pose {
        children [
          Shape {
            appearance BrushedAluminium {
              colorOverride 0.85098 0.85098 0.85098
            }
            geometry Mesh {
              url [
                "Plate1.stl"
              ]
            }
          }
        ]
      }
    ]
    name IS name
    boundingObject Pose {
      translation 0 0 0.001
      children [
        Box {
          size 0.3 0.05 0.002
        }
      ]
    }
    physics Physics {
      density -1
      mass 0.001
    }
  }
}
